product-group-common/src/commonMain/kotlin/models/PrgrpGroup.kt

```diff
@@ -0,0 +1,10 @@
+package com.crowdproj.marketplace.product.group.common.models
+
+data class PrgrpGroup(
+    var id: PrgrpGroupId = PrgrpGroupId.NONE,
+    var name: String = "",
+    var description: String = "",
+    var ownerId: PrgrpUserId = PrgrpUserId.NONE,
+    var propertiesFkId: PrgrpPropertiesId = PrgrpPropertiesId.NONE,
```

Comment:
Думаю, здесь нужно что-то такое
```suggestion
    var properties: MutableSet<PrgrpPropertyId> = mutableSetOf(),
```